tools/bin/hawqregister

```diff
@@ -729,6 +728,23 @@ class HawqRegister(object):
                 for k, eof in enumerate(eofs[1:]):
                     query += ',(%d, %d, %d, %d, %d)' % (self.firstsegno + k + 1, eof, -1, -1, -1)
```

Comment:
This is minor, but I believe the `%` syntax is being [deprecated](https://www.python.org/dev/peps/pep-3101/) in Python 3.x. Maybe consider replacing it with the new `.format` syntax.